### DISCUSSION TITLE
Next update

### DISCUSSION THREAD

**Discussion by Idunnoman001** [3kh0/ext-remover#265] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Will the next update use bookmarklets becasue my school district sadly blocked them.

**Comment by CheeseKingz333** [comment]
> Will the next update use bookmarklets becasue my school district sadly blocked them.

SAAAAAME EVEN THE HTML PHILLIPBRUH MADE DOESN'T WORK I JUST WANNA MAKE MY ANIMATIONS AND THE SIDE IS BLOCKED FRRR

**Reply by CheeseKingz333** [reply]
Whens It Coming out?

**Reply by Idunnoman001** [reply]
they said in around a month im pretty sure

**Reply by CheeseKingz333** [reply]
> they said in around a month im pretty sure

ah Ok

**Reply by Idunnoman001** [reply]
not sure you heard the news but the original dev cant fix it so the next update probably wont come out

**Reply by md4r2** [reply]
[Pingot](https://github.com/Holding-THE-CREATOR/Pingot) is coming in around a month.

**Reply by 3kh0** [reply]
I dont think pingot is even gonna work

**Comment by ILikeBypassing** [comment]
just wait.

**Comment by ItzJoseD** [comment, marked as answer]
Hey, I read a few comments and it seems like we are all in a "I want to unblock this" situation, so i'm giving you guys access to my proxies, (Premium TN ones too!) Here are the links,

https://womginx1.lunarvcoding.repl.co/

https://myholyunblockerv5o5.herokuapp.com/

https://ultrapromaxplus.herokuapp.com/

And thats it, enjoy this internet freedom, (You can also log in with ur personal G-Mail!)

**Reply by 3kh0** [reply]
they are not premium tn links but ok
